### PR TITLE
Add icon selector component

### PR DESCRIPTION
**Pull request by RobertMartinis**
image

### CODE REVIEW

File: app/src/components/AddIngredient/AddIngredient.tsx [hunk 1]
```diff
@@ -36,10 +36,15 @@ const AddIngredient = ({ handleIngredientAdd }: AddIngredientProps) => {
     if (data) setOptions(data.data.searchResults);
   };
 
+  const addIngredientClick = () => {
+    if (ingredient === "" || ingredient === "" || ingredient === "") return;
```
Comment (RobertMartinis) [inline app/src/components/AddIngredient/AddIngredient.tsx:40]: LOL!

Comment (JakobPaulsson) [reply app/src/components/AddIngredient/AddIngredient.tsx:40]: wtf

Comment (RobertMartinis) [inline app/src/components/AddIngredient/AddIngredient.tsx:40]:
```suggestion
    if (ingredient === "" || amount === "" || unit === "") return;
```

**Review by RobertMartinis — COMMENTED** [review]
Fix

**Comment by JakobPaulsson** [comment]
Tung

**Comment by JakobPaulsson** [comment]
based

**Comment by JakobPaulsson** [comment]
fuck missclick

**Pull request merged**